src/vopr.zig

```diff
@@ -1179,7 +1179,7 @@ pub const Simulator = struct {
                 v.value_ptr.set(replica.replica);
 
                 log.debug("{}: core_missing_blocks: " ++
-                    "missing address={} checksum={} corrupt={} (remote read)", .{
+                    "missing address={} checksum={x:0>32} corrupt={} (remote read)", .{
```

Comment:
I'm not entirely sure about hexing checksums here in the simulator as they come from the blocks, not message headers. Happy to revert changes in this file!